.circleci/config.yml

```diff
@@ -0,0 +1,120 @@
+# .circleci/config.yml
+version: 2.1
+
+executors:
+  osx-arm64:
+    macos:
+      xcode: 14.2.0 # indicate your selected version of Xcode
+    resource_class: macos.m1.large.gen1
+  linux-aarch64:
+    machine:
+      image: ubuntu-2004:current
+    resource_class: arm.medium
+
+jobs: # a basic unit of work in a run
+  build_and_test:
+    parameters:
+      os:
+        type: executor
+    executor: << parameters.os >>
+    steps:
+      - checkout
+
+      - run:
+          name: Fetch bioconda install script
+          command: wget https://raw.githubusercontent.com/bioconda/bioconda-common/master/install-and-set-up-conda.sh
+
+      - restore_cache:
+          keys:
+            - bioconda-utils-{{ arch }}-{{ checksum "install-and-set-up-conda.sh" }}
+
+      - run:
+          name: Install bioconda-utils
+          command: |
+            sudo mkdir -p /opt/miniconda
+            sudo chown -R $USER /opt/miniconda
+            bash install-and-set-up-conda.sh
```

Comment:
```
ERROR: File or directory already exists: '/opt/miniconda'
If you want to update an existing installation, use the -u option.

Exited with code exit status 1
```
[1] https://app.circleci.com/pipelines/github/Yikun/bioconda-recipes/11/workflows/42683084-4273-4ec5-8f25-f5d6f1b6be29/jobs/17

I hitted above issue, and resolved by change [install-and-set-up-conda.sh#L42](https://github.com/bioconda/bioconda-common/blob/e03cb8697f45928e918a029429d96befc352e2b1/install-and-set-up-conda.sh#L42):
```patch
- bash miniconda.sh -b -p "${MINICONDA_INSTALLATION_DIR}"
+ bash miniconda.sh -u -b -p "${MINICONDA_INSTALLATION_DIR}"
```

After this change, it works!